automatic/bleachbit.install/tools/chocolateyInstall.ps1

```diff
@@ -1,13 +1,16 @@
-﻿$toolsDir = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
+﻿$ErrorActionPreference = 'Stop'
+
+$toolsDir = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
+$Installer = (Get-ChildItem $toolsDir -filter "*.exe").FullName
 
 $packageArgs = @{
-  packageName   = $env:ChocolateyPackageName
-  softwareName  = "BleachBit"
-  file          = "$toolsDir\BleachBit-2.0-setup.exe"
-  fileType      = "exe"
-  silentArgs    = "/S"
+   packageName   = $env:ChocolateyPackageName
+   file          = $Installer
+   fileType      = "exe"
+   silentArgs    = "/S /allusers"
+   validExitCodes= @(0)
 }
 
 Install-ChocolateyInstallPackage @packageArgs
```

Comment:
Installer should be deleted after installation. It is already present in nupkg file itself.